# This is literally bug fix, not port patch.
# I have requested a pull to upstream, will remove it
# when it is merged.

diff --git a/src/steps/os/unix.rs b/src/steps/os/unix.rs
index 52df093..61cefc7 100644
--- a/src/steps/os/unix.rs
+++ b/src/steps/os/unix.rs
@@ -135,7 +135,7 @@ pub fn run_fish_plug(ctx: &ExecutionContext) -> Result<()> {
     ctx.run_type().execute(&fish).args(&["-c", "plug update"]).check_run()
 }
 
-#[cfg(not(target_os = "macos"))]
+#[cfg(not(any(target_os = "android", target_os = "macos")))]
 pub fn upgrade_gnome_extensions(ctx: &ExecutionContext) -> Result<()> {
     let gdbus = require("gdbus")?;
     require_option(
